### PR TITLE
Updated deps, reran build, locked ava at 0.17

### PR DESCRIPTION
**Pull request by cwonrails**
Ava left at 0.17.0 due to issues with tests identified elsewhere.

### CODE REVIEW

**Comment by cwonrails** [comment]
If the AVA version bump here would lead to problems upstream let me know and I'll revert.

**Comment by johnotander** [comment]
🔥

File: build.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 const fs = require('fs')
 const glob = require('glob')
 const cssScss = require('css-scss')
-const stream = require('stream');
```
Comment (johnotander) [inline build.js]: 💟

Comment (cwonrails) [inline build.js]: Standard 4eva!

**Pull request merged**